### PR TITLE
haskell: add helm-hoogle, bind to mhf

### PR DESCRIPTION
**Pull request by jb55**
Accidently deleted the branch, so I had to create a new PR D:

### CODE REVIEW

**Comment by robbyoconnor** [comment]
:+1:

File: layers/+lang/haskell/packages.el [hunk 1]
```diff
@@ -144,6 +150,7 @@
         "hd"  'inferior-haskell-find-haddock
         "hh"  'hoogle
         "hH"  'hoogle-lookup-from-local
+        "hf"  'helm-hoogle
```
Comment (joehillen) [inline layers/+lang/haskell/packages.el]: Might want to check for helm before creating this binding as well.

Comment (jb55) [inline layers/+lang/haskell/packages.el]: gah! thanks for catching this

**Comment by syl20bnr** [comment]
Thank you :+1: 
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**